ci-operator/jobs/openshift-ossm/maistra/openshift-ossm-maistra-postsubmits.yaml

```diff
@@ -0,0 +1,502 @@
+postsubmits:
+  maistra/envoy:
```

Comment:
Please remove all of the presubmits. Try to `make jobs` in order to generate the correct one. In order to have postsubmits you will need to add a `promotion` configuration in the ci-operator config.